### PR TITLE
Add finale tour examples and tests

### PR DESCRIPTION
**Pull request by didrocks**
* Fix existing example by stamping 2.10 as a string.
* Add finale tour examples and new tests for those.

They can be enabled as they are only pulling from npm and wiki parts, which are already tested elsewhere, so should work under CI jenkins firewall rules.

### CODE REVIEW

**Comment by sergiusens** [comment]
retest this please

**Comment by fgimenez** [comment]
retest this please

**Comment by fgimenez** [comment]
retest this please

**Comment by sergiusens** [comment]
I approve of this. Just wish the tests passed.

@elopio or @fgimenez I hope you can Shepperd this in.

**Comment by sergiusens** [comment]
retest this please

File: snaps_tests/__init__.py [hunk 1]
```diff
@@ -160,9 +161,11 @@ def install_snap(self, snap_content_dir, snap_name, version):
             self.addCleanup(
                 self.snappy_testbed.run_command,
                 ['rm', snap_path_in_testbed])
+            cmd = ['sudo', 'snap', 'install', snap_path_in_testbed]
+            if devmode:
+                cmd.append("--devmode")
```
Comment (elopio) [inline snaps_tests/__init__.py]: single quotes :)
Thanks for the addition.

File: snaps_tests/tour_tests/00-SNAPCRAFT/test_01_easy_start.py [hunk 2]
```diff
@@ -25,6 +25,6 @@ class EasyStartTestCase(snaps_tests.SnapsTestCase):
     def test_hello(self):
         self.skipTest('Skip while the proxy rule is added by IS.')
```
Comment (elopio) [inline snaps_tests/tour_tests/00-SNAPCRAFT/test_01_easy_start.py]: The rule for gnu.org was supposedly added already. I haven't tried it yet.

**Comment by elopio** [comment]
I approve too. I will fight with the tests after the meeting.

File: snaps_tests/__init__.py (hunk 1 above)
Comment (didrocks) [inline snaps_tests/__init__.py]: rohhhhh :) ok, just pushed that change

**Pull request merged**